backend/python/app/tests/services/test_story_service.py

```diff
@@ -79,9 +119,65 @@ def test_create_translation_raises_error_if_translation_content_commit_fails_and
+    translator_obj = db_session_add_commit_obj(
+        db, UserAll(first_name="Translatorfirst", last_name="Translatorlast", email=f"{translator_authId}", is_deleted=False, auth_id=translator_authId)
+    )
+
+    reviewer_authId = ''.join(random.choice(alphanumeric) for i in range(10))
+    reviewer_obj = db_session_add_commit_obj(
+        db, UserAll(first_name="Reviewerfirst", last_name="Reviewerlast", email=f"{reviewer_authId}", is_deleted=False, auth_id=reviewer_authId)
+    )
+
+    story_obj = db_session_add_commit_obj(
+        db, Story(title="Title", description="Description", youtube_link="", level=1)
+    )
+
+    story_translation_obj = db_session_add_commit_obj(
+        db, StoryTranslation(story_id=story_obj.id, language="English", stage="TRANSLATE", translator_id=translator_obj.id, reviewer_id=reviewer_obj.id)
+    )
+
+    story_translation_content_obj_1 = db_session_add_commit_obj(
+        db, StoryTranslationContent(story_translation_id=story_translation_obj.id, line_index=0, translation_content="Translation content 1.", status="APPROVED")
+    )
+
+    story_translation_content_obj_2 = db_session_add_commit_obj(
+        db, StoryTranslationContent(story_translation_id=story_translation_obj.id, line_index=1, translation_content="", status="DEFAULT")
+    )
+
+    translation_contents = [
+        {
+            "id": story_translation_content_obj_1.id,
+            "line_index": story_translation_content_obj_1.line_index,
+            "translation_content": story_translation_content_obj_1.translation_content,
+            "status": story_translation_content_obj_1.status,
+        }, {
+            "id": story_translation_content_obj_2.id,
+            "line_index": story_translation_content_obj_2.line_index,
+            "translation_content": story_translation_content_obj_2.translation_content,
+            "status": story_translation_content_obj_2.status,
+        },
+    ]
+
+    resp = services["story"].get_story_translation(story_translation_obj.id)
+
+    assert resp["id"] == story_translation_obj.id
+    assert resp["language"] == story_translation_obj.language
+    assert resp["stage"] == story_translation_obj.stage
+    assert resp["translation_contents"] == translation_contents
+    assert resp["translator_id"] == story_translation_obj.translator_id
+    assert resp["reviewer_id"] == story_translation_obj.reviewer_id
+    assert resp["story_id"] == story_translation_obj.story_id
+    assert resp["title"] == story_obj.title
+    assert resp["description"] == story_obj.description
+    assert resp["youtube_link"] == story_obj.youtube_link
+    assert resp["level"] == story_obj.level
+    assert resp["num_translated_lines"] == services["story"]._get_num_translated_lines(translation_contents)
+    assert resp["num_approved_lines"] == services["story"]._get_num_approved_lines(translation_contents)
+    assert resp["num_content_lines"] == len(translation_contents)
 
```

Comment:
Didn't assert `resp["translator_name"]` or `resp["reviewer_name"]` because they aren't returned in `get_story_translation`, but rather `get_story_translations`.